tests/docs/expected.js

```diff
@@ -115,9 +115,9 @@ module.exports = {
     'cacheFor',
     'camelize',
     'canCatalogEntriesByType',
-    'canInvoke',
     'cancel',
```

Comment:
This file was partially sorted, but not fully. I decided to sort it to make it consistent.